### PR TITLE
Added boolean flag for testnet

### PR DESCRIPTION
**Pull request by kwek20**
Required for changes in iotaledger/iri#1295
Should not be merged before this PR is merged

### CODE REVIEW

File: configs/iri-pod.j2 [hunk 1]
```diff
@@ -22,9 +22,11 @@ spec:
         - "--snapshot"
         - "/iri/data/snapshot.txt"
         - "--testnet"
+        - "true"
         - "--testnet-coordinator"
         - "BTCAAFIH9CJIVIMWFMIHKFNWRTLJRKSTMRCVRE9CIP9AEDTOULVFRHQZT9QAQBZXXAZGBNMVOOKTKAXTB"
         - "--testnet-no-coo-validation"
+        - "true"
```
Comment (GalRogozinski) [inline configs/iri-pod.j2]: I think it will be nicer if the arg will be in the same line as the config

**Comment by GalRogozinski** [comment]
has this been tested?

File: configs/iri-pod.j2 (hunk 1 above)
Comment (kwek20) [reply configs/iri-pod.j2]: That would be not according to the other arguments/format

**Comment by kwek20** [comment]
@GalRogozinski yes @DyrellC ran it locally

File: configs/iri-pod.j2 (hunk 1 above)
Comment (GalRogozinski) [reply configs/iri-pod.j2]: :+1:

**Pull request merged**